scripts/ci/update_release.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+import yaml
+from typing import Dict
+
+RELATIVE_PATH_TO_MANAGER_YAML = "config/manager/manager.yaml"
+RELATIVE_PATH_TO_OPENSHIFT_MANAGER_YAML = "deploy/openshift/operator_openshift.yaml"
+
+
+def _load_yaml_file(path: str) -> Dict:
+    with open(path, "r") as f:
+        return yaml.safe_load(f.read())
+
+
+def _dump_yaml(operator: Dict, path: str) -> None:
```

Comment:
return type is not None

Reply:
you are correct! Thanks for catching that